diff --git a/node_modules/osmtogeojson/index.js b/node_modules/osmtogeojson/index.js
index 0164459..4d2f9f6 100644
--- a/node_modules/osmtogeojson/index.js
+++ b/node_modules/osmtogeojson/index.js
@@ -3,7 +3,9 @@ var rewind = require("@mapbox/geojson-rewind");
 
 // see https://wiki.openstreetmap.org/wiki/Overpass_turbo/Polygon_Features
 var polygonFeatures = {};
-require("osm-polygon-features").forEach(function(tags) {
+var osmPolygonFeatures = require("osm-polygon-features");
+
+osmPolygonFeatures.forEach(function(tags) {
   if (tags.polygon === "all")
     polygonFeatures[tags.key] = true;
   else {
